inbm-lib/inbm_common_lib/validater.py

```diff
@@ -36,6 +36,22 @@ def validate_date(date: str) -> str:
     except ValueError:
         raise argparse.ArgumentTypeError(f"Not a valid date - format YYYY-MM-DD: '{date}")
 
+def validate_guid(value: str, param_type: str, max_size: int) -> str:
```

Comment:
You'll need to add some unit tests to ensure this is working as it should.  Positive and negative.

Reply:
@nmgaston @Siva472 also please check that it's following the guid format:

xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx 

Each 'x' is a hex digit: 0,1,2,3,4,5,6,7,8,9,a,A,b,B,c,C,d,D,e,E,f,F

See http://guid.one/guid